src/ORM/DataExtension.php

```diff
@@ -84,18 +114,37 @@ public function populateDefaults()
     {
     }
 
+
     public function can($member)
     {
     }
 
+    /**
+     * Whether or not the given $member is able to edit this decorated object.
+     *
+     * @param  Member $member
+     * @return bool
+     */
     public function canEdit($member)
     {
     }
 
+    /**
+     * Whether or not the given $member is able to delete this decorated object.
+     *
+     * @param  Member $member
+     * @return bool
```

Comment:
Technically these methods return `@return int|bool` (see `Permission::check()`). The DataObject equivalents of these methods don't indicate that though, which is why it's a bit misleading